src/intercom/front_end_binding.py

```diff
@@ -34,11 +34,15 @@ def get_available_analysis_plugins(self):
         if plugin_file is not None:
             plugin_dict = pickle.loads(plugin_file.read())
             return plugin_dict
-        raise Exception("No available plug-ins found. FACT backend might be down!")
+        raise Exception('No available plug-ins found. FACT backend might be down!')
 
     def get_binary_and_filename(self, uid):
         return self._request_response_listener(uid, 'raw_download_task', 'raw_download_task_resp')
 
+    def peek_in_binary(self, uid: str, offset: int, length: int) -> bytes:
+        ''' Return part of the content from file `uid` from offset `offset` with length `length`. '''
```

Comment:
```suggestion
```

1. I think this comment is not required, as the code is self-explanatory :-)
2. codacy complains here about the whitespaced surrounding of this docstring

Reply:
I think you are right => removed the comment